packages/hoppscotch-common/src/helpers/import-export/import/openapi.ts

```diff
@@ -929,16 +931,40 @@ export const hoppOpenAPIImporter = (fileContents: string[]) =>
 
             for (const docObj of docArr) {
               try {
-                const isValidOpenAPISpec = isABasicOpenAPIDoc(docObj)
+                console.log(
+                  "Validating OpenAPI spec:",
+                  JSON.stringify(docObj).substring(0, 100) + "..."
+                )
+
+                // More lenient check - if it has paths, we'll try to import it
+                const isValidOpenAPISpec =
```

Comment:
most of these validations are already in the `isABasicOpenAPIDoc` function. you can add any additional validations into that function only.

or you can remove the `isABasicOpenAPIDoc` function, its unused if you're doing the validation here.